src/test/java/org/apache/sysds/test/functions/jmlc/ReuseModelVariablesTest.java

```diff
@@ -35,7 +35,7 @@
 public class ReuseModelVariablesTest extends AutomatedTestBase 
 {
 	private final static String TEST_NAME1 = "reuse-glm-predict";
-	private final static String TEST_NAME2 = "reuse-msvm-predict";
+	private final static String TEST_NAME2 = "reuse--predict";
```

Comment:
why?

Reply:
accident i guess, while i searched for msvm.